tailor-app/src/main/java/com/future/tailormade/tailor_app/core/model/ui/orderDetail/OrderDetailMeasurementUiModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.future.tailormade.tailor_app.core.model.ui.orderDetail
+
+data class OrderDetailMeasurementUiModel(
+
+    var chest: String? = null,
+
```

Comment:
in float

Reply:
This value has been mapped to String in UiModel to set the text directly kak
Any concern kak?

Reply:
Has been resolved